src/web_socket.toit

```diff
@@ -189,14 +208,28 @@ class WebSocket:
     current_reader_ = null
 
   /**
-  Abruptly closes the WebSocket.
-  Does not send a close packet first.
+  May abruptly close the WebSocket.
+  If we are in a suitable place in the protocol, we send a close packet first.
   */
-  close:
+  close --status_code/int=STATUS_WEBSOCKET_NORMAL_CLOSURE:
+    close_write --status_code=STATUS_WEBSOCKET_NORMAL_CLOSURE
     socket_.close
-    current_writer_ = null
     current_reader_ = null
 
+  /**
+  May abruptly close the WebSocket.
+  If we are in a suitable place in the protocol, we send a close packet first.
+  */
+  close_write --status_code/int=STATUS_WEBSOCKET_NORMAL_CLOSURE:
```

Comment:
When do I need to call which?
Do I need to call `close_write` after/before calling `close`.
Is it necessary to call `close` after a `close_write`?
...
Add more comments.